### PR TITLE
Add email when vol deselected

### PR DESCRIPTION
**Pull request by brandtheisey**
This sends an email to a volunteer if they are deselected / deleted.

Uses text provided by @ultrasaurus .

Repalces #742 

Address #709

### CODE REVIEW

File: api/models/Volunteer.js [hunk 1]
```diff
@@ -45,6 +45,33 @@ module.exports = {
     };
 
     noteUtils.notifier.notify(params, cb);
-  }
+  },
 
+  beforeDestroy: function (values, cb){
+   Volunteer.findOne({ id: values.where.id }).exec(function(err, volunteer) {
+      if (err) done(err);
```
Comment (dhcole) [inline api/models/Volunteer.js]: @brandtheisey Should this be `cb(err)` instead of `done(err)`?

**Comment by dhcole** [comment]
One comment about; otherwise looks good to me!

File: api/models/Volunteer.js (hunk 1 above)
Comment (brandtheisey) [inline api/models/Volunteer.js]: You are correct, nice catch. Is fixed now.

**Comment by dhcole** [comment]
:ship: thanks @brandtheisey

**Pull request merged**